src/main/ruby/core/exception.rb

```diff
@@ -236,6 +236,17 @@ class NotImplementedError < ScriptError
 end
 
 class Interrupt < SignalException
+  def initialize(*args)
+    case args.size
+    when 0
+      message = nil
+    when 1
+      message = args.first
+    else
+      raise ArgumentError, "wrong number of arguments (#{args.size} for 0..1)"
+    end
```

Comment:
Can you also use required/optional arguments here instead of manually checking args.size?

Reply:
Yes, fixed.